pkg/autoupdate/autoupdate.go

```diff
@@ -0,0 +1,80 @@
+// autoupdate is a convenience package to implement autoupdates in libpod. While
+// the logic is implemented in libpod, the package provides an idiomatic and
+// coherent way to define available policies.
+package autoupdate
+
+import (
+	"sort"
+
+	"github.com/containers/image/v5/docker"
+	"github.com/containers/image/v5/docker/reference"
+	"github.com/containers/image/v5/transports/alltransports"
+	"github.com/pkg/errors"
+)
+
+// Label denotes the container/pod label key to specify auto-update policies.
+const Label = "io.containers.autoupdate"
+
+type Policy string
+
+const (
+	// PolicyDefault is the default policy denoting no auto updates.
+	PolicyDefault Policy = "default"
```

Comment:
I would give the default policy a more explicit name, something like `disabled`

Reply:
Good idea!